ui/src/app/job-details/tabs/scattered-attempts/scattered-attempts.component.html

```diff
@@ -0,0 +1,62 @@
+<h2 mat-dialog-title *ngIf="data">{{ data.shardsData.taskName }}</h2>
+<mat-dialog-content #dialogContainer *ngIf="data && data.shardsData">
+  <mat-accordion class="content">
+    <mat-expansion-panel disabled="disabled" expanded="false" class="list-header">
+      <mat-expansion-panel-header>
+        <mat-panel-title><div class="task-name">Task Name</div></mat-panel-title>
+        <mat-panel-description>
+          <div class="task-status">Status</div>
+          <div class="task-start">Start</div>
+          <div class="task-duration">Duration</div>
+          <div class="task-inputs">Inputs</div>
+          <div class="task-outputs">Outputs</div>
+          <div class="task-links">Links</div>
+          <div class="task-attempts">Attempts</div>
+        </mat-panel-description>
+      </mat-expansion-panel-header>
+    </mat-expansion-panel>
+    <ng-container *ngFor="let shard of data.shardsData.taskShards">
+      <mat-expansion-panel class="list-row" expanded="false" (opened)="getShardAttempts(shard)">
+        <mat-expansion-panel-header [expandedHeight]="'48px'">
+          <mat-panel-title>
+            <div class="task-name">shard {{ shard.shardIndex }}</div>
+          </mat-panel-title>
+          <mat-panel-description>
+            <div class="task-status status-icon">
+              <clr-tooltip>
+                <clr-icon clrTooltipTrigger [attr.shape]="getStatusIcon(shard.executionStatus)" size="18"></clr-icon>
+                <clr-tooltip-content clrPosition="left" clrSize="xs" *clrIfOpen>
+                  <span>{{ shard.executionStatus }}</span>
+                </clr-tooltip-content>
+              </clr-tooltip>
+            </div>
+            <div class="task-start">
+              <jm-datetime [datetime]="shard.start"></jm-datetime>
+            </div>
+            <div class="task-duration">
```

Comment:
This looks like its actually a repeated div multiple times. It may be worth abstracting to a component?